src/image-button/image-button.style.tsx

```diff
@@ -21,29 +21,31 @@ export const Main = styled.button`
 
     img {
         width: 3.5rem;
-    }
-
-    ${MediaQuery.MaxWidth.tablet} {
-        padding: 1.5rem 0rem;
+        height: 3.5rem;
     }
 
     ${MediaQuery.MaxWidth.mobileL} {
-        padding: 1rem 0rem;
+        padding: 0.5rem 0.25rem;
     }
 
-    ${MediaQuery.MinWidth.tablet} {
-        :focus:active {
-            box-shadow: ${DesignToken.ElevationBoxShadow};
-            ${TextStyleHelper.getTextStyle("H4", "bold")}
-        }
-
-        @media (hover: hover) {
```

Comment:
is this media query still needed?

Reply:
removed.

Reply:
i believe long presses do trigger hover and tend to "stick" if the user doesn't explicitly tap on something else, but it's purely a cosmetic issue